.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         julia-version: ['1.5']
         julia-arch: [x64]
-        os: [ubuntu-latest, macOS-latest, windows-latest]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
i think we're good testing in ubuntu and windows..

Reply:
so do i